azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -271,10 +273,15 @@ public void setGlobalProps(Props globalProps) {
     // Every 2 mins clean the recently finished list
     private static final long RECENTLY_FINISHED_INTERVAL_MS = 2 * 60 * 1000;
 
+    // Every 5 mins kill flows running longer than allowed max running time
+    private static final long LONG_RUNNING_FLOW_KILLING_INTERVAL_MS = 5 * 60 * 1000;
```

Comment:
Agreed.
My comment was more directed against using primitives instead of dedicated classes for temporal variables in our internal code.

Regarding configuration parameters, `Duration` can be there as well. There are methods like [`java.time.Duration#parse`](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-) which uses the  ISO-8601 duration format `PnDTnHnMn.nS`. Example:
```
# Set max flow running time to 15 min
azkaban.server.max.flow.running=PT15M
```
That being said, I totally agree with you in retaining existing configuration. This may be useful if/when we create new ones.